tubearchivist/static/script.js

```diff
@@ -381,6 +381,19 @@ function deleteVideo(button) {
   }, 1000);
 }
 
+function deleteAndIgnoreVideo(button) {
+  let to_delete = button.getAttribute('data-id');
+  let to_redirect = button.getAttribute('data-redirect');
+  let endpointDelete = '/api/video/' + to_delete + '/';
+  apiRequest(endpointDelete, 'DELETE');
+  let endpoingDownload = '/api/download/';
```

Comment:
Here and on the line below shouldn't it be endpoin**t**Download instead of endpoing?

Reply:
It definitely should.